recipe/meta.yaml

```diff
@@ -11,38 +11,35 @@ source:
   sha256: d1877192b44d36055f39fe212ca0abf04416bac1ff81be1b6cda0fbe82c12ac5
 
 build:
-  number: 0
+  number: 2
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
-    - python >=3.6.2
+    - python >=3.7
   run:
-    - addict >=2.4.0
-    - jsonschema ==3.2.0
+    - jsonschema >=3.2.0
     - jstyleson >=0.0.2
     - natsort >=7.1.0
-    - networkx >=2.5
-    - ninja >=1.10.0.post2
-    - numpy >1.19.2
+    - networkx >=2.6, <=2.8.2
+    - ninja >=1.10.0.post2, <1.11
+    - numpy >1.19.2, <1.25
+    - openvino-telemetry >=2023.1.0
+    - packaging >=20.0
+    - pandas >=1.1.5, <2.1
+    - pip
+    - pip:
+        - pymoo @ git+https://github.com/anyoptimization/pymoo.git@695cb26923903f872c7256a9013609769f3cc2bd
+    - psutil
     - pydot >=1.4.1
-    - python >=3.6.2
+    - pyparsing <3.0
+    - python >=3.7
     - scikit-learn >=0.24.0
-    - scipy >=1.3.2
-    # - scipy >=1.3.2  # [py>=37]
-    # - scipy >=1.3.2,<=1.5.4  # [py<37]
-    - matplotlib-base
-    # - matplotlib-base >=3.3.4  # [py>=37]
-    # - matplotlib-base >=3.3.4,<3.4  # [py<37]
-    - pandas
-    # - pandas >=1.1.5  # [py>=37]
-    # - pandas >=1.1.5,<1.2  # [py<37]
+    - scipy >=1.3.2, <1.11
     - texttable >=1.6.3
-    # - pytorch >=1.5.0,<=1.8.1,!=1.8.0
     - tqdm >=4.54.1
-    - wheel >=0.36.1
     # - {{ compiler('c') }}
     # - {{ compiler('cxx') }}
```

Comment:
please, also enable `pip check`:
https://github.com/conda-forge/nncf-feedstock/blob/faaa29c0f86977f6c7478c0c1717b8ebfb02c631/recipe/meta.yaml#L54-L55
to ensure that dependency set is valid

Reply:
Done